podman rmi --ignore (#13541)
containers/podman: docs/source/markdown/podman-rmi.1.md

This option will cause podman to remove all containers that are using the image before removing the image from the system.

#### **--ignore**, **-i**

> Ignore if a specified image does not exist in the local storage and do not throw an error.

Reviewer: ```suggestion
If a specified image does not exist in the local storage, ignore the error and continue processing.
```

Author: "continue processing" isn't accurate. `rmi` always continues processing but throws errors at the end. With --ignore, such errors will not be reported.

Author: Rephrased but without the "continue" part :heavy_check_mark:
